tools/wake/main.cpp

```diff
@@ -578,17 +581,29 @@ int main(int argc, char **argv) {
   std::unique_ptr<Top> top(new Top);
   std::vector<ExternalFile> wakefiles;
   wakefiles.reserve(wakefilenames.size());
-  for (auto &i : wakefilenames) {
-    if (verbose && debug) std::cerr << "Parsing " << i << std::endl;
 
-    wakefiles.emplace_back(terminalReporter, i.c_str());
+  bool alerted_slow_cache = false;
+
+  for (size_t i = 0; i < wakefilenames.size(); i++) {
+    auto &wakefile = wakefilenames[i];
+    auto now = std::chrono::steady_clock::now();
+    if (std::chrono::duration_cast<std::chrono::milliseconds>(now - start).count() > 1000) {
+      std::cout << "Scanning " << i << "/" << wakefilenames.size()
+                << " wake files. Cache may be cold.\r" << std::flush;
```

Comment:
Please say that its the "kernel file cache" to shift blame away from wake. People may assume that "cache" is a wake thing and will gripe at us if they do.